src/wiki/help-pages/Change-Themes.md

```diff
@@ -0,0 +1,135 @@
+---
+eleventyNavigation:
+  key: Change Themes
+  parent: Help pages
+--- 
+
+# How to change your Themes in Prism Launcher
+
+Themes can help you spice up your launcher appearance
+
+Prism Launcher 6.0 will add support for multiple custom themes at once
+
+***This section will be updated accordingly***
+
+Some themes are only on discord for now, but we will try to transfer them to our repository.
+
+Prism Launcher use Qt stylesheet, to make themes please look at [Qt Documentation](https://doc.qt.io/qt-6/stylesheet-reference.html).
+
+## Themes Downloads
+
+[![Prism Themes Repository](https://img.shields.io/badge/Prism--Themes--Repository-76AE88?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes)
+
+### Themes Currently Available
+
+[![Breeze Dark](https://img.shields.io/badge/Breeze--Dark-blue?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Breeze-Dark.zip)
+[![Catppuccin Frappe](https://img.shields.io/badge/Catppuccin--Frappe-EF9F76?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Frappe.zip)
+[![Catppuccin Latte](https://img.shields.io/badge/Catppuccin--Latte-DC8A78?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Latte.zip)
+[![Catppuccin Macchiato](https://img.shields.io/badge/Catppuccin--Macchiato-A6DA95?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Macchiato.zip)
+[![Catppuccin Mocha](https://img.shields.io/badge/Catppuccin--Macchiato-DDB6F2?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Catppuccin-Mocha.zip)
+[![Dracula](https://img.shields.io/badge/Dracula-BD93F9?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Dracula.zip)
+[![Fluent Dark](https://img.shields.io/badge/Fluent--Dark-60CDFF?style=for-the-badge&logo=github&logoColor=333333)](https://github.com/PrismLauncher/Themes/releases/latest/download/Fluent-Dark.zip)
+[![Nord](https://img.shields.io/badge/Nord-88C0D0?style=for-the-badge&logo=github&logoColor=white)](https://github.com/PrismLauncher/Themes/releases/latest/download/Nord.zip)
+
+### Themes Available only on Discord for now
+
+[![Borest Theme](https://img.shields.io/badge/Borest--Theme-1E1E2E?style=for-the-badge&logo=discord&logoColor=white)](https://discord.com/channels/1031648380885147709/1033569391931047957)
```

Comment:
We should add this themes to prismlauncher/themes instead

Reply:
I will ask the theme author